package/kernel/linux/modules/crypto.mk

```diff
@@ -646,9 +659,33 @@ endef
 ifndef CONFIG_TARGET_x86_64
   define KernelPackage/crypto-misc/x86
     FILES+=$(LINUX_DIR)/arch/x86/crypto/twofish-i586.ko
+    FILES+=$(LINUX_DIR)/arch/x86/crypto/serpent-sse2-i586.ko
+    FILES+=$(LINUX_DIR)/arch/x86/crypto/glue_helper.ko
+    FILES+=$(LINUX_DIR)/crypto/ablk_helper.ko
+    FILES+=$(LINUX_DIR)/crypto/cryptd.ko
+    FILES+=$(LINUX_DIR)/crypto/gf128mul.ko
+    FILES+=$(LINUX_DIR)/crypto/lrw.ko
+    FILES+=$(LINUX_DIR)/crypto/xts.ko
   endef
```

Comment:
why not use multi line with \

Reply:
I figured there just weren't very many there, so it wasn't worth rewriting the existing line. The difference is cosmetic anyway.